ui/library/dist/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@allenai/pdf-components",
-    "version": "0.0.9",
+    "version": "0.1.0",
```

Comment:
This change should be in a separate branch/PR dedicated to republishing a new version of the lib.

Reply:
@carolinepaulic okey i will put it back to normal and separate to another PR once this is merged